modules/ossm-about-revisionbased-strategy.adoc

```diff
@@ -0,0 +1,6 @@
+:_mod-docs-content-type: Concept
+[id="about-revisionbased-strategy"]
+= About RevisionBased strategy
+:context: ossm-about-revisionbased-strategy
+
+Typically, an update that is performed using  the `RevisionBased` strategy runs two revisions of a control plane. This capability allows you to gradually migrate workloads from the old control plane to the new control plane, making canary upgrades possible. The 'RevisionBased' strategy also allows you to update by more than one minor version.
```

Comment:
are the ticks wrong for the second occurrence of RevisionBased ?

Reply:
done